public_html/script.js

```diff
@@ -1391,3 +1599,57 @@ function getAirframesModeSLink(code) {
 
     return "";   
 }
+
+
+// takes in an elemnt jQuery path and the OL3 layer name and toggles the visibility based on clicking it
+function toggleLayer(element, layer) {
+	// set initial checked status
+	ol.control.LayerSwitcher.forEachRecursive(layers, function(lyr) { 
+		if (lyr.get('name') === layer && lyr.getVisible()) {
+			$(element).addClass('settingsCheckboxChecked');
+		}
+	});
+	$(element).on('click', function() {
+		var visible = false;
+		if ($(element).hasClass('settingsCheckboxChecked')) {
+			visible = true;
+		}
+		ol.control.LayerSwitcher.forEachRecursive(layers, function(lyr) { 
+			if (lyr.get('name') === layer) {
+				if (visible) {
+					lyr.setVisible(false);
+					$(element).removeClass('settingsCheckboxChecked');
+				} else {
+					lyr.setVisible(true);
+					$(element).addClass('settingsCheckboxChecked');
+				}
+			}
+		});
+	});
+}
+
+// check status.json if it has a serial number for a flightfeeder
+function flightFeederCheck() {
+	$.ajax('/status.json', {
+		success: function(data) {
+			if (typeof data.serial !== 'undefined') {
```

Comment:
I updated the FF status.json in 7.6.0; there is now a `"type": "flightfeeder"` key that is probably better to check than looking for the serial number (uncommissioned FFs won't have a serial number)